src/widgets/radio.jsx

```diff
@@ -49,88 +194,62 @@ var BaseRadio = React.createClass({
                     <$_>Select all that apply.</$_>
                 </div>}
             {this.props.choices.map(function(choice, i) {
+                var classSet = { "inline": !this.props.onePerLine };
 
-                var content = <div>
-                        {choice.content}
-                    </div>;
+                // True if we're in review mode and a clue is available
+                var reviewModeClues = rubric && rubric.choices[i].clue;
+                var showClue = choice.checked &&
+                    (exerciseClues || reviewModeClues);
+
+                var element = Choice;
+                var elementProps = {
+                    ref: `radio${i}`,
+                    checked: choice.checked,
+                    clue: choice.clue,
+                    content: choice.content,
+                    disabled: this.props.apiOptions.readOnly,
+                    groupName: radioGroupName,
+                    showClue: showClue,
+                    type: inputType,
+                    onChecked: (checked) => {
+                        this.checkOption(i, checked);
+                    }
+                }
+
+                if (choice.isNoneOfTheAbove) {
+                    element = ChoiceNoneAbove;
+                    _.extend(elementProps, { showContent: choice.correct });
```

Comment:
This is kinda weird: I know this could be easily written as
```javascript
elementProps.showContent = choice.correct
```
but I think that by using `_.extend` the code is much more understandable for another dev. Still, I'm using a third party library to do something that wouldn't require it, so it's up to you if you prefer efficiency over readability or the other way around :)